DigitalLearningSolutions.Data/Services/DelegateUploadFileService.cs

```diff
@@ -240,22 +238,22 @@ private void RegisterDelegate(DelegateTableRow delegateRow, DateTime? welcomeEma
             }
         }
 
-        private void UpdateUserProfessionalRegistrationNumberIfNecessary(bool? delegateRowHasPrn, string? delegateRowPrn, int delegateId, bool isUpdate)
+        private void UpdateUserProfessionalRegistrationNumberIfNecessary(bool? delegateRowHasPrn, string? delegateRowPrn, int delegateId)

```

Comment:
Nitpicking, but this line is a bit long. Running the auto formatter in Rider breaks a couple lines in this file.